cmd/config/docs/api-conventions/functions-spec.md

```diff
@@ -327,16 +327,22 @@ A function SHOULD preserve comments when input serialization format is YAML.
 This allows for human authoring of configuration to coexist with changes made by
 functions.
 
-### Annotations
-
-The orchestrator annotates resources in the wire format with annotation prefix
-`config.kubernetes.io`. These annotations are not persisted when the
-orchestrator writes the resources to the filesystem. The orchestrator sets this
-annotation when reading files from the local filesystem and removes the
-annotation when writing the output of functions back to the filesystem.
-
-In general, a function MUST NOT modify these annotations except the ones
-explicitly listed below.
+### Control Annotations
+
+For orchestration purposes, the orchestrator will use a set of annotations,
+referred to as _control annotations_, on resources in `Resources.items`. These
+annotations are not persisted to resource manifests on the filesystem: The
+orchestrator sets this annotation when reading files from the local filesystem
+and removes the annotation when writing the output of functions back to the
+filesystem.
+
+Annotation prefix `fn-ctrl.config.kubernetes.io` is reserved for use for control
```

Comment:
should this be namespaced by controller?

Reply:
I suspect the word "control" (and "ctrl") here may be overloaded and causing confusion. There's no concept of controller in this context. "control annotations" is defined as annotations that ONLY exist on the wire and are not persisted to manifests. Open to renaming this.

Reply:
A few alternate suggestions:

`internal.config.kubernetes.io`
`tools.config.kubernetes.io`

Reply:
How about calling these "Orchestrator Annotations"?

`orchestrator.config.kubernetes.io`

"internal" and "tools" are not very descriptive.

Reply:
I slightly prefer `internal.config.kubernetes.io` -- I agree that it is very generic and doesn't indicate where the annotation comes from, but it does communicate that (a) if you didn't add it, you probably shouldn't touch it and (b) if it ends up in the external output, something is wrong. Arguably those are the most important characteristics to convey? I also like `orchestrator.config.kubernetes.io` as an alternative though.

Reply:
Ok, let's go with `internal`.